docs/index.md

```diff
@@ -3,4 +3,33 @@ layout: home
 nav_order: 1
 ---
 
-# BeSecure Schema
+### Be-Secure Schema Definitions
+
+An easy-to-use data format that maps precisely to various data exchange needs between cybersecurity analysts and security labs. These schema definitions address many of the shortcomings of dealing with vulnerabilities in open source software and ML models in Be-Secure ecosystem.
+
+These unified format means that vulnerability databases, open source users, and security researchers can easily share tooling and consume vulnerabilities across all of open source. This means a more complete view of vulnerabilities in open source for everyone, as well as faster detection and remediation times resulting from easier automation.
+
+Key Be-Secure assets these schema definitions cater to are as below.
```

Comment:
reword BeS Assets as BeS Eco System Entities

Reply:
Change made as per the feedback